app/features/recent-list/components/RecentList.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+import { push } from 'react-router-redux';
+
+import { ConferenceCard, RecentListContainer, TruncateText } from '../styled';
+
+type Props = {
+
+    /**
+     * Redux dispatch.
+     */
+    dispatch: Dispatch<*>;
+
+    /**
+     * Array of recent conferences.
+     */
+    _recentList: Array<Object>;
```

Comment:
Cool. `type Conference` coming soon.

Reply:
Since this is a recent list, how about some `RecentListItem` instead? We don't really have a `Conference` type, and adding it for this sounds a bit odd to me.